kernel/arch/dreamcast/hardware/spu.c

```diff
@@ -226,17 +256,19 @@ void spu_memset_sq(uintptr_t dst, uint32_t what, size_t length) {
 void spu_reset_chans(void) {
     int i;
     g2_fifo_wait();
-    g2_write_32(SNDREGADDR(0x2800), 0);
+    g2_ctx_t ctx = g2_lock();
+    g2_write_32_raw(SNDREGADDR(0x2800), 0);
 
     for(i = 0; i < 64; i++) {
-        if(!(i % 4)) g2_fifo_wait();
+        if((i & 3) == 0) g2_fifo_wait();
```

Comment:
Cosmetic changes to their own commit.

Reply:
This is performance change ;)
Division with remainder takes more CPU time than just AND.

Reply:
Again here GCC will emit the exact same code so the change is pointless.

Reply:
Okay, I'll know.